geode-redis/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -14,28 +14,44 @@
  */
 package org.apache.geode.redis.internal;
 
+import org.apache.geode.cache.PartitionAttributesFactory;
 import org.apache.geode.cache.Region;
 import org.apache.geode.cache.RegionShortcut;
 import org.apache.geode.internal.cache.InternalCache;
 import org.apache.geode.internal.cache.InternalRegionFactory;
+import org.apache.geode.management.ManagementException;
 import org.apache.geode.redis.internal.data.ByteArrayWrapper;
 import org.apache.geode.redis.internal.data.RedisData;
+import org.apache.geode.redis.internal.executor.cluster.RedisPartitionResolver;
 
 public class RegionProvider {
   /**
    * The name of the region that holds data stored in redis.
    */
-  private static final String REDIS_DATA_REGION = "__REDIS_DATA";
-  private static final String REDIS_CONFIG_REGION = "__REDIS_CONFIG";
+  public static final String REDIS_DATA_REGION = "__REDIS_DATA";
+  public static final String REDIS_CONFIG_REGION = "__REDIS_CONFIG";
+  public static final String REDIS_REGION_BUCKETS_PARAM = "redis.region.buckets";
+  public static final int REDIS_REGION_BUCKETS =
+      Integer.getInteger(REDIS_REGION_BUCKETS_PARAM, 128);
+  public static final int REDIS_SLOTS = Integer.getInteger("redis.slots", 16384);
```

Comment:
Comments said you made this fixed but it is still configurable.

Reply:
Oops - thanks!